browser-app/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "name": "browser-app",
+  "version": "0.0.0",
+  "dependencies": {
+    "@theia/core": "latest",
```

Comment:
we should not use 'latest' as for CQs we're only validating against specific versions 
versions may be in yarn.lock but by using a specific version here ensure any developer is using correct version as well

Reply:
fixed